File: cci05.06-convert-integer-lcci/javascript/Solution.js
/**
 * @param {number} A
 * @param {number} B
 * @return {number}
 */
var convertInteger = function(A, B) {
    let C = A ^ B
    let count = 0
    for (let i = 0; i < 32; ++i) {
        if ((C&1) == 1) {
            count++
        }
        C >>= 1
    }
    return count
};

File: cci05.06-convert-integer-lcci/cpp/Solution.cpp
class Solution {
public:
    int convertInteger(int A, int B) {
        int C = A ^ B;
        int count = 0;
        for (int i = 0; i < 32; ++i) {
            if ((C & 1) == 1) {
                count++;
            }
            C >>= 1;
        }
        return count;
    }
};
